clusterloader2/cmd/clusterloader.go

```diff
@@ -95,6 +95,8 @@ func validateClusterFlags() *errors.ErrorList {
 }
 
 func initFlags() {
+	// Set the default logging level to 2"
+	flags.Set("v", "2")
```

Comment:
I don't think that's the right way to go - see:
https://github.com/kubernetes/perf-tests/pull/1271#issuecomment-633546818

Reply:
@wojtek-t  Sorry, I didn't see your comment. I'm not familiar with test-infra. Does it make sense to set v(2) in  [clusterloader2/api/default.go](https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/api/default.go), which is called by [clusterloader2/pkg/test.go](https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/pkg/test/test.go)?

```
func (conf *Config) SetDefaults() {
       flags.Set("V", defaultLogLevel)
   ...
 }
```

Reply:
@wojtek-t  Sorry, I didn't see your comments.  I agree we need more thoughts on how to manage the verbosity level. 

Since we just wanted to remove the detailed measurement logging information, instead of setting the v levels, we can use the the existing field _EnableLogging_ , which is already defined in struct _WaitForPodOptions_ (_WaitForNodeOptions_, _WaitForPVSOptions_, and _WaitForPVCSOptions_). The current version always sets it  true. The PR adds a new flag _--enable-measurement-logging_ and sets the value of _EnableLogging_ according to the flag. 
```
type WaitForPodOptions struct {
    Selector            *ObjectSelector
    DesiredPodCount     int
    EnableLogging       bool
    CallerName          string
    WaitForPodsInterval time.Duration
}
```